java/openjdk17/files/JDK-8266242-undecl-ident-nsbun-arm64-workaround.patch

```diff
@@ -0,0 +1,20 @@
+--- src/java.desktop/macosx/native/libawt_lwawt/awt/CGraphicsDevice.m.orig	2024-10-08 18:09:38.000000000 +1100
++++ src/java.desktop/macosx/native/libawt_lwawt/awt/CGraphicsDevice.m	2024-10-08 18:09:16.000000000 +1100
+@@ -60,6 +60,9 @@
+ }
+ 
+ static BOOL isValidDisplayMode(CGDisplayModeRef mode) {
++    // https://trac.macports.org/ticket/71049: temporary additional guard for undef'd NSBun..ARM64
++    // Should be reported/fixed upstream at openjdk.org.
```

Comment:
It has already been reported, right? https://bugs.openjdk.org/browse/JDK-8266242

Reply:
The original report was for a problem with later os versions >= 11.00. This is a regression from the backporting of that fix; the original is considered fixed. I guess they might re-open instead; not sure how openjdk run their bug systems.

Reply:
I have opened a new issue, but haven't got an issue ID back yet.

Reply:
Ah, ok.